docs/errors/db.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Database errors
+description: An index of Clerk errors related to database operations.
+type: reference
+---
+
+# Database errors 
+
+An index of Clerk errors related to database operations.
+
+## `DuplicateAllowlistIdentifier`
+
+```json
+{
+    "shortMessage": "duplicate allowlist identifier",
+    "longMessage":  "the identifier <identifier> already exists",
+    "code":         "duplicate_record_code"
+}
+```
```

Comment:
I doubt this is the correct error meant for this page. This needs to get moved to the allowlist identifiers page, no?
Or does DB stand for something else, and I was mistaken to change it to "Database"?

Reply:
was there anyone you were working with on team sdk to get these docs out ? if not, would you mind reaching out to them to get this sorted ?

Reply:
Some of these errors will never get sent out to the frontend or backend. This is probably one of them, and is the correct error.